src/argilla/utils/dependency.py

```diff
@@ -58,36 +58,43 @@ def _compare_versions(
         )
     if not ops[op](version.parse(got_version), version.parse(want_version)):
         raise ImportError(
-            f"{requirement} must be installed{f' to use `{func_name}`' if func_name else ''}, but found {package}=={got_version}."
+            f"{requirement} must be installed{f' to use `{fn_name}`' if fn_name else ''}, but found {package}=={got_version}."
             f" You can install a supported version of '{package}' with this command: `pip install -U {requirement}`"
         )
 
 
-def require_version(requirement: str, func_name: Optional[str] = None) -> None:
+def require_dependencies(requirements: Union[str, List[str]], fn_name: Optional[str] = None) -> None:
     """
     Perform a runtime check of the dependency versions, using the exact same syntax used by pip.
     The installed module version comes from the *site-packages* dir via *importlib_metadata*.
 
     Args:
-        requirement (`str`): pip style definition, e.g.,  "tokenizers==0.9.4", "tqdm>=4.27", "numpy"
+        requirements: pip style definition of dependencies, e.g. "tokenizers==0.9.4", "tqdm>=4.27", "numpy". Can either be a single dependency or a list of dependencies.
         func_name (`str`, *optional*): what suggestion to print in case of requirements not being met
 
     Example:
     ```python
-    require_version("pandas>1.1.2")
-    require_version("datasets>1.17.0", "from_datasets")
+    require_dependencies("pandas>1.1.2")
+    require_dependencies("datasets>1.17.0", "from_datasets")
+    require_dependencies(["pandas>1.1.2", "datasets>1.17.0"])
     ```
     """
 
+    if isinstance(requirements, list):
+        if len(requirements) == 0:
+            return
+        require_dependencies(requirements[0], func_name=fn_name)
+        return require_dependencies(requirements[1:], func_name=fn_name)
```

Comment:
I have no real preference here, but we could also just loop over the requirements and call them with each element rather than this head-tails approach.
E.g.
```python
for requirement in requirements:
    require_dependencies(requirement, fn_name=fn_name)
```